package.json

```diff
@@ -113,7 +113,7 @@
     "stylelint": "^13.5.0",
     "stylelint-config-recommended-scss": "^4.1.0",
     "ts-loader": "^8.0.2",
-    "typescript": "^4.0.2",
+    "typescript": "^3.9.7",
```

Comment:
Was there any specific reason for this downgrade? TS v4 is not the lts version and we should use it.